projects/ngrx.io/content/guide/effects/index.md

```diff
@@ -23,13 +23,20 @@ In a service-based application, your components interact with data through many
 
 Imagine that your application manages movies. Here is a component that fetches and displays a list of movies.
 
+
 <code-example header="movies-page.component.ts">
+
+import { Component, inject } from '@angular/core';
```

Comment:
If we're importing `inject` here I think it's also good to update the snippet to use inject instead of using the constructor. 
Do you also want to change this, or should we pick this up in a different PR?

Note: the example of `<code-example header="movies.service.ts">` also needs to be updated using `injet`